### PR TITLE
cmd: support pause/resume/remove changefeed 

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

* Support pause/resume/remove changefeed via cdc cli.
* Add status address to capture info.

Cc https://github.com/pingcap/ticdc/issues/542

### Check List 

Tests 

 - Integration test

Code changes

 - Has exported variable/fields change

Related changes

 - Need to cherry-pick to the release branch

### Release note

- Support pause/resume/remove changefeed via cdc cli.

### CODE REVIEW

File: cdc/http_status.go [hunk 1]
```diff
@@ -51,13 +52,20 @@ func (s *Server) startStatusHTTP() {
 
 	addr := fmt.Sprintf("%s:%d", s.opts.statusHost, s.opts.statusPort)
 	s.statusServer = &http.Server{Addr: addr, Handler: serverMux}
-	log.Info("status http server is running", zap.String("addr", addr))
+	errCh := make(chan error)
 	go func() {
-		err := s.statusServer.ListenAndServe()
+		ln, err := net.Listen("tcp", addr)
```
Comment (leoppro) [inline cdc/http_status.go:57]: move this line out of this goroutine? we can avoid to using error channel

**Comment by overvenus** [comment]
/run-integration-tests

PTAL, thanks

File: cmd/server.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func init() {
 	rootCmd.AddCommand(serverCmd)
 
 	serverCmd.Flags().StringVar(&serverPdAddr, "pd", "http://127.0.0.1:2379", "PD address, separated by comma")
-	serverCmd.Flags().StringVar(&statusAddr, "status-addr", "0.0.0.0:8300", "Bind address for http status server")
+	serverCmd.Flags().StringVar(&statusAddr, "status-addr", "127.0.0.1:8300", "Bind address for http status server")
```
Comment (amyangfei) [inline cmd/server.go:37]: DBA suggests setting the default value for `status host` to `0.0.0.0`, the same with TiDB

Comment (overvenus) [reply cmd/server.go:37]: 0.0.0.0 in capture info is meaningless. 127.0.0.1 makes user assign a meaningful address.

**Comment by amyangfei** [comment]
btw, as this PR changes `status-addr` to `addr`, we also need to update parameter in [tipocket](https://github.com/pingcap/tipocket/blob/master/pkg/test-infra/cdc/recommend.go#L95) and [cdc document](https://github.com/pingcap/docs-cn/blob/master/reference/tools/ticdc/deploy.md)

**Pull request merged**